Content/src/Fable.MyLib.fsproj

```diff
@@ -1,16 +1,13 @@
-<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Version>0.0.0</Version>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
It's better for libs to target the lower framework.
As long as we don't use any specific feature of netstandard2.0.

Here you will end up to force users to update their project even if it's not really needed.

Reply:
Thanks for the clarification! I'm still getting my head around .net core 2.0 and version number overload.